common/script/public/userServices.js

```diff
@@ -107,10 +107,14 @@ angular.module('habitrpg')
                       if (err) {
                         var message = err.code ? err.message : err;
                         console.log(message);
```

Comment:
Not an issue in this PR, but while we're stomping around here--is there a reason we spit all of these into the console? It's always struck me as kinda odd. @crookedneighbor